internal/appv2/patches.go

```diff
@@ -243,6 +244,18 @@ func ensureArrayOfMap(raw any) ([]map[string]any, error) {
 	return out, nil
 }
 
+func patchMounts(cfg map[string]any) (map[string]any, error) {
+
```

Comment:
I find it weird but lately common that people adds a newline after the function signature and so many blank lines between bodies which are so small already. Is this caused by some editor (like vscode), some go formatting tool or just added manually?

Reply:
honestly i can't tell for sure if i did that unintentionally or vscode auto inserted those spaces. 
I've cleaned it up. Can i get a review(✅) on this?